From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Tue, 16 Feb 2021 19:10:28 -0500
Subject: Ship python examples with python3 in shebang line

See discussion at #982630 and
https://www.python.org/dev/peps/pep-0394/#recommendation :

> When packaging third party Python scripts, distributors are
> encouraged to change less specific shebangs to more specific
> ones. This ensures software is used with the latest version of
> Python available, and it can remove a dependency on Python
> 2. The details on what specifics to set are left to the
> distributors; though. Example specifics could include:
>
> - Changing python shebangs to python3 when Python 3.x is
>   supported.

Forwarded: not-needed
---
 lang/python/examples/assuan.py                   | 2 +-
 lang/python/examples/decryption-filter.py        | 2 +-
 lang/python/examples/delkey.py                   | 2 +-
 lang/python/examples/exportimport.py             | 2 +-
 lang/python/examples/genkey.py                   | 2 +-
 lang/python/examples/inter-edit.py               | 2 +-
 lang/python/examples/low_level-encrypt_to_all.py | 2 +-
 lang/python/examples/sign.py                     | 2 +-
 lang/python/examples/signverify.py               | 2 +-
 lang/python/examples/simple.py                   | 2 +-
 lang/python/examples/testCMSgetkey.py            | 2 +-
 lang/python/examples/verifydetails.py            | 2 +-
 12 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/lang/python/examples/assuan.py b/lang/python/examples/assuan.py
index 84f5ff0..317f5b5 100755
--- a/lang/python/examples/assuan.py
+++ b/lang/python/examples/assuan.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 #
diff --git a/lang/python/examples/decryption-filter.py b/lang/python/examples/decryption-filter.py
index d0fc3a4..99f4240 100755
--- a/lang/python/examples/decryption-filter.py
+++ b/lang/python/examples/decryption-filter.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016, 2018 g10 Code GmbH
 #
diff --git a/lang/python/examples/delkey.py b/lang/python/examples/delkey.py
index f0411f4..823f384 100755
--- a/lang/python/examples/delkey.py
+++ b/lang/python/examples/delkey.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2004,2008 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/exportimport.py b/lang/python/examples/exportimport.py
index 235d570..9c811e8 100755
--- a/lang/python/examples/exportimport.py
+++ b/lang/python/examples/exportimport.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2004,2008 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/genkey.py b/lang/python/examples/genkey.py
index 5df6476..bbc03a3 100755
--- a/lang/python/examples/genkey.py
+++ b/lang/python/examples/genkey.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2004 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/inter-edit.py b/lang/python/examples/inter-edit.py
index 2fd74c2..e89d55a 100755
--- a/lang/python/examples/inter-edit.py
+++ b/lang/python/examples/inter-edit.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2005 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/low_level-encrypt_to_all.py b/lang/python/examples/low_level-encrypt_to_all.py
index edd0197..7e9c6f4 100755
--- a/lang/python/examples/low_level-encrypt_to_all.py
+++ b/lang/python/examples/low_level-encrypt_to_all.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2008 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/sign.py b/lang/python/examples/sign.py
index 079ccde..4c9e587 100755
--- a/lang/python/examples/sign.py
+++ b/lang/python/examples/sign.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2002 John Goerzen <jgoerzen@complete.org>
diff --git a/lang/python/examples/signverify.py b/lang/python/examples/signverify.py
index c9c6edd..8cf7932 100755
--- a/lang/python/examples/signverify.py
+++ b/lang/python/examples/signverify.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2004,2008 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/simple.py b/lang/python/examples/simple.py
index d2e7c07..2defb01 100755
--- a/lang/python/examples/simple.py
+++ b/lang/python/examples/simple.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2005 Igor Belyi <belyi@users.sourceforge.net>
diff --git a/lang/python/examples/testCMSgetkey.py b/lang/python/examples/testCMSgetkey.py
index 9b5a526..c5b1f21 100755
--- a/lang/python/examples/testCMSgetkey.py
+++ b/lang/python/examples/testCMSgetkey.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2008 Bernhard Reiter <bernhard@intevation.de>
diff --git a/lang/python/examples/verifydetails.py b/lang/python/examples/verifydetails.py
index 3733486..1bf2e48 100755
--- a/lang/python/examples/verifydetails.py
+++ b/lang/python/examples/verifydetails.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2016 g10 Code GmbH
 # Copyright (C) 2004,2008 Igor Belyi <belyi@users.sourceforge.net>
